### ISSUE TITLE
not support websites builded with 'Flutter'?

### ISSUE THREAD

**Issue by SungJun-Jo** [nolanlawson/fuite#16]
When I run fuite to a site built with 'flutter', I only receive 'No tests to run.' messages.

Is there a way to test with the "flutter" site?

**Comment by nolanlawson** [comment]
Does Flutter not have any internal links? I.e. `<a href="/foo">`? Or does it use some other accessible format, like `<div role="link">`?

**Comment by bolshoytoster** [comment]
@nolanlawson it appears flutter application run inside a `<canvas>`. You can see in this [sample site](gallery.flutter.dev) at `/html/body/flt-glass-pane//flt-scene-host/flt-scene/flt-canvas-container/canvas`. I'm not sure how you'd get around that.

**Comment by nolanlawson** [comment]
Yeah, it looks like they have a button you can click to "enable accessibility." When I click it, it fills the DOM with a bunch of `<flt-semantics-value>` elements. But even those elements don't have the proper role for internal links –the role is "generic" when it should be ["link"](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_link_role). They do have the right "button" role for the Settings button, though.

![Screenshot from 2021-12-21 09-18-37](https://user-images.githubusercontent.com/283842/146971873-0d92a9c7-58a1-44f5-938a-fdc319e69edb.png)

I think in order for fuite to support this out-of-the-box, Flutter should at least have the proper "link" role. But even in that case, fuite would not be able to identify the link (e.g. `"/contact"` or `"/about"`). But it could click it, presumably.

As a side note, I think most accessibility advocates would be appalled at how Flutter does it. :sweat_smile:
